src/lightmap.cpp

```diff
@@ -492,6 +492,46 @@ bool map::pl_line_of_sight( const tripoint &t, const int max_range ) const
 
 #include "messages.h"
 
+typedef ShadowcastingSlope slope;
+
+slope::ShadowcastingSlope( int x, int y )
```

Comment:
I tried doing slope::slope but it didn't work. ShadowcastingSlope::ShadowcastingSlope is probably better in this case.